README.rst

```diff
@@ -16,6 +16,14 @@ If you haven't yet enabled multi-factor authentication for AWS API access, check
 
 Installation:
 -------------
+
+.. code-block:: sh
+
+    $ brew update
```

Comment:
May be worth pointing at https://reddit.atlassian.net/wiki/spaces/IO/pages/122494415/Configuring+Homebrew+to+use+the+Reddit+Tap+Repository to keep these up to date. Notice how we're already missing the recently added tap-pin.